source/_components/feedreader.markdown

```diff
@@ -23,11 +23,26 @@ feedreader:
     - https://www.home-assistant.io/atom.xml
     - https://github.com/blog.atom
     - https://hasspodcast.io/feed/podcast
+  scan_interval:
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

Reply:
No worries, I'll add a separate configuration example.